src/main/java/com/okta/sdk/models/log/Log.java

```diff
@@ -0,0 +1,180 @@
+package com.okta.sdk.models.log;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * @author Guy Zehavi
+ */
+public final class Log {
+
+    private final LogResource actor;
```

Comment:
It should extends from ApiObject and need comments for all the fields and methods.